src/com/company/Power.java

```diff
@@ -1,28 +1,28 @@
 package com.company;
-import java.util.Scanner; // импорт сканера
+import java.util.Scanner; 
 
 public class Power {
 
     static Scanner scan = new Scanner(System.in);
 
-    public int getInt(){//валидатор ввода данных
+    public int getInt(){
         int num;
         if(scan.hasNextInt()){
             num = scan.nextInt();
         } else {
             System.out.println("Вы допустили ошибку при вводе числа. Попробуйте еще раз.");
             scan.next();
-            num = getInt();//рекурсия
+            num = getInt();
         }
         return num;
     }
 
-    static int getPower(int Num, int Pow){ // Num - число, Pow - степень
-        int result = 1;//число в 0 степени = 1
-        for (int i=1; i<=Pow; i++){//в цикле умножаем число Num количество Pow раз само на себя
+    static int getPower(int Num, int Pow){ 
```

Comment:
fields names are written in lowerCamelCase 
https://mate-academy.github.io/style-guides/java/java.html#s5.1-identifier-names

Reply:
use more understandable names of params